1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,57 @@
 function highlightWords(paragraph, colours) {
-  // Write your code here...
+  // Create a `<p>` element.
+  const pElement = document.createElement("p");
+
+  // Create a `<select>` element.
+  const myDropdown = document.createElement("select");
```

Comment:
Issue: The select needs a default option that tells the user to select a colour, if you check the tests you can find the text that it's expecting for this option.